composer.json

```diff
@@ -25,17 +25,17 @@
         "ext-hash": "*",
         "ext-json": "*",
         "ext-tokenizer": "*",
-        "clue/ndjson-react": "^1.0",
+        "clue/ndjson-react": "^1.3",
         "composer/semver": "^3.4",
         "composer/xdebug-handler": "^3.0.5",
         "fidry/cpu-core-counter": "^1.2",
         "react/child-process": "^0.6.6",
-        "react/event-loop": "^1.0",
-        "react/promise": "^2.11 || ^3.0",
```

Comment:
Is this change required for the project?
We stuck on promise v2 in our open source project because there are a lot of breaking changes and undesirable behavior in v3. And now it will be impossible to update CS fixer